google-cloud-storage/src/test/java/com/google/cloud/storage/StorageImplTest.java

```diff
@@ -424,6 +425,13 @@ private void initializeServiceDependentObjects() {
     expectedBucket3 = new Bucket(storage, new BucketInfo.BuilderImpl(BUCKET_INFO3));
   }
 
+  @Test
+  public void testInvocationIDDisabledBucket() {
```

Comment:
What does `Bucket` mean in this test name?

Reply:
typo!